stable/elasticsearch-exporter/values.yaml

```diff
@@ -84,4 +84,8 @@ serviceMonitor:
   ## https://github.com/coreos/prometheus-operator
   ##
   enabled: false
+  namespace: ""
```

Comment:
Would it be more elegant if namespace is commented out, and the template checks for its existence?

Reply:
Yes, I am with you. I commented out the `namespace` property. There is already a check for its existence in the `servicemonitor.yaml` file